src/NuGet.Clients/NuGet.VisualStudio.Implementation/NuGet.VisualStudio.Implementation.csproj

```diff
@@ -29,8 +29,8 @@
     <ProjectReference Include="..\NuGet.VisualStudio.Common\NuGet.VisualStudio.Common.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Build" IncludeAssets="None" PrivateAssets="all"/>
-    <PackageReference Include="Microsoft.Build.Framework" IncludeAssets="None" PrivateAssets="all"/>
+    <PackageReference Include="Microsoft.Build" IncludeAssets="None" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.Build.Framework" IncludeAssets="None" PrivateAssets="all" NoWarn="NU1605" />
```

Comment:
What's the downgrade we're suppressing here?

This is not your change, but do you know why we have an IncludeAssets="None", PrivateAssets="all" package? Is it forcing some specific resolution?

Reply:
Some other package depends on 17.14 MSBuild and we reference 17.11.